pouta_blueprints/static/partials/navbar.html

```diff
@@ -24,8 +24,11 @@
   </div>
   <div ng-show="isLoggedIn()" class="navbar-form navbar-right">
     <form ng-submit="logout()">
-        <button type="submit" class="btn btn-success hidden-sm">Log out {{ getUserName() }}</button>
+        <button type="submit" class="btn btn-success hidden-sm">Log out</button>
```

Comment:
Remove *-sm classes and then remove the other button as they are the same from now on